app/middlewares/schemaValidator.js

```diff
@@ -0,0 +1,12 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+const { schemaError } = require('../errors');
+
+module.exports = schema => [
+  checkSchema(schema),
+  (req, _, next) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) return next(schemaError(errors));
```

Comment:
are returning just one error here?

Reply:
Mmm, I don't understand. we review?

Reply:
solved with .array()[0]

I will remember this security tip!

Reply:
![Screenshot from 2021-07-16 09-08-27](https://user-images.githubusercontent.com/87040068/125961138-d18dcbb2-9640-4901-bce1-d597d775d635.png)